### ISSUE TITLE
Update `pallet-contracts` dependency

### ISSUE THREAD

**Issue by ebma** [pendulum-chain/pendulum#224]
As pointed out by our tech advisors, there was a bug in the `pallet-contracts` dependency which could be exploited by malicious actors. The dependency is already fixed upstream and we just need to use the latest version, ie. run 
```
cargo update -p pallet-contracts
```

**Comment by TorstenStueber** [comment]
We would also need to deploy this on Amplitude and Pendulum once this is merged. However, no ticket is required as there will be scheduled runtime upgrades anyway.

**Comment by b-yap** [comment]
dumb question:  if i remember,  this command only going to update the cargo.lock. Is that the only plan? and is updating the actual toml files in another ticket?

**Comment by ebma** [comment]
Yes @b-yap, that's what I imagined this ticket to do. 

But our tech advisors actually pointed out that we might need to do some migration for our storage because the storage version of the contracts pallet is not set properly. That should probably get done in another ticket.
